lib/ansible/modules/cloud/amazon/iam_cert.py

```diff
@@ -269,10 +276,21 @@ def main():
     cert_chain = module.params.get('cert_chain')
     dup_ok = module.params.get('dup_ok')
     if state == 'present':
-        cert = open(module.params.get('cert'), 'r').read().rstrip()
-        key = open(module.params.get('key'), 'r').read().rstrip()
+        try:
+            cert = open(module.params.get('cert'), 'r').read().rstrip()
+        except IOError:
```

Comment:
I love the idea of having input from a file or string for this module, although I'm curious if this best method or if we should have 2 separate vars similar to copy etc.  eg.  mutually exclusive vars cert_path and cert_content

I too would like to see this implemented soon along with [my PR for iam_cert](https://github.com/ansible/ansible/pull/20622)